deptofdefense/solo
# Feat 110 add worker task definition and ECS service for running celery worker

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist
I have…
~~[ ] attached screenshots illustrating relevant behavior before and after my changes.~~
- [x] review and update SSAG documentation if needed.

## Summary of Changes
This pull request…
- closes #110 closes #111
- added container definition template for the celery worker
- added task definition to terraform
- added service to existing ECS cluster for worker
- added cloudwatch log group to be used by the worker

## Testing
To verify the changes proposed in this pull request…
1. Verify terraform successfully creates staging worker service and task definition
2. Verify worker service connects to RDS for use as message queue

## Screenshots
N/A
